app/helpers/tasks/import/dwca/psu_import_helper.rb

```diff
@@ -1,3 +1,35 @@
 module Tasks::Import::Dwca::PsuImportHelper
 
+  def text_row(row)
```

Comment:
Eventually, perhaps: The name makes it consistent with the controller and HTML file names as per convention.